dev/io.openliberty.servlet.internal.6.0.factories/src/io/openliberty/servlet/internal/v60/osgi/srt/factory/SRTConnectionContextPool60Impl.java

```diff
@@ -0,0 +1,48 @@
+/*******************************************************************************
+ * Copyright (c) 2011, 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.servlet.internal.v60.osgi.srt.factory;
+
+import org.osgi.service.component.annotations.Component;
+
+import com.ibm.ws.webcontainer.osgi.srt.SRTConnectionContextPool;
+import com.ibm.ws.webcontainer40.osgi.srt.SRTConnectionContext40;
+
+/**
+ * A simple pool for SRTConnectionContext31 objects.
```

Comment:
remove this comment

Reply:
Oops, another copy/paste error. I'll fix that. Thanks for catching it.